python/tvm/topi/cuda/scatter.py

```diff
@@ -787,42 +787,45 @@ def gen_ir(data_ptr, indices_ptr, updates_ptr, out_ptr):
         for i in data_ptr.shape:
             fused_shape *= i
 
-        # For now we avoid parallizing over dimensions indexed by `indices` as
-        # there may be repeated indices and hadling parallel accumulation can
-        # be hard. So we parallelize over X_M .. X_{N-1} instead. This will
-        # work well when these dimensions are large enough to saturate memory
-        # bandwidth, but performance will be bad when these dimensions are
-        # small.
```

Comment:
Can you add a comment about how we are doing parallelism (we are thread-parallel over all the update dimension and each block handles one set of indices?)

Reply:
We follow the original parallelism scheme, but replace `ib.for_range()` with `blockIdx.y`.
`Atomic_add` guarantees correctness when `mode="add"`

Reply:
Can you update the comment in the code to reflect this?

Reply:
Added